packages/widgetbook_annotation/README.md

```diff
@@ -0,0 +1,180 @@
+[![Discord](https://img.shields.io/discord/879618555560218625?color=blue&style=flat-square)](https://discord.com/invite/zT4AMStAJA)
+[![style: very good analysis](https://img.shields.io/badge/style-very_good_analysis-B22C89.svg?style=flat-square)](https://pub.dev/packages/very_good_analysis)
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/widgetbook/widgetbook/ci?style=flat-square)
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/widgetbook/widgetbook/ci?label=test&style=flat-square)
+
+
+# Introduction 
+
+This package contains annotations for [package:widgetbook_generator](https://pub.dev/packages/widgetbook_generator) with which the generator will create the Widgetbook defined in [package:widgetbook](https://pub.dev/packages/widgetbook). Therefore, this package is a part of making [package:widgetbook](https://pub.dev/packages/widgetbook) easier to maintain. Furthermore, setting up and maintaining [package:widgetbook](https://pub.dev/packages/widgetbook) is simplified by code generation. 
+
+# Installing this package
+This package requires the following dependencies: 
+
+| Package           | Pub |
+| ----------------- | --------------------------------- |
+| [package:widgetbook](https://pub.dev/packages/widgetbook) | [![Pub Version](https://img.shields.io/pub/v/widgetbook?style=flat-square)](https://pub.dev/packages/widgetbook) |
+| [package:widgetbook_annotation](https://pub.dev/packages/widgetbook_annotation) | [![Pub Version](https://img.shields.io/pub/v/widgetbook_annotation?style=flat-square)](https://pub.dev/packages/widgetbook_annotation) |
+
+and the following dev dependencies:
+
+| Package           | Pub |
+| ----------------- | --------------------------------- |
+| [package:widgetbook_generator](https://pub.dev/packages/widgetbook_generator) | [![Pub Version](https://img.shields.io/pub/v/widgetbook_generator?style=flat-square)](https://pub.dev/packages/widgetbook_generator) |
+| [package:build_runner](https://pub.dev/packages/build_runner) | [![Pub Version](https://img.shields.io/pub/v/build_runner?style=flat-square)](https://pub.dev/packages/build_runner) |
+
+The `pubspec.yaml` file could look like this:
+
+```
+dependencies:
+  widgetbook:
+  widgetbook_annotation:
+
+dev_dependencies:
+  build_runner:
+  widgetbook_generator:
+```
+
+# Avaialble annotations
+
+This package defines the annotations `WidgetbookApp`, `WidgetbookStory`, and `WidgetbookTheme`. The annotations and their usage are explained below.
+
+## WidgetbookApp
+
+The annotation `WidgetbookApp` has to be set only once and is mandatory for the code generation process. It not not important which element is annotated, but the location of the file in which `WidgetbookApp` is used defines the folder in which [package:widgetbook_generator](https://pub.dev/packages/widgetbook_generator) will create the file `app.widgetbook.main` file. The `app.widgetbook.main` file requires all the code to run the Widgetbook. 
```

Comment:
It is not important

Reply:
in which WidgetbookApp is used, defines

Reply:
will create the file app.widgetbook.main --> delete last word "file"

Reply:
The `app.widgetbook.main` file requires all the code to run the Widgetbook.  --> I don't get the meaning of this sentence

Reply:
requires is wrong. should be 'The `app.widgetbook.main` file conains all the code to run the Widgetbook'